### ISSUE TITLE
Allow users to view identify results for stacked parcels

### ISSUE THREAD

**Issue by rkelson** [agrc/parcels#131]
In some counties condo and apartment parcels might be a stack of parcels with the same geometry. In Utah county, south of the Vineyard Connector Rd., there are good examples of this:
https://parcels.utah.gov/#/location/-12440666.900157992,4910819.476840503,1128.497176
![image](https://user-images.githubusercontent.com/6200941/206519116-422a119b-03fd-4b9a-b356-fbd21817cf0d.png)

I can see Utah Co is dealing with this a few different ways however, _stacked_ parcels exist. It would be an improvement if the app allowed users to click through the multiple parcels with identical geometries in the identify results:
![image](https://user-images.githubusercontent.com/6200941/206520342-71eedc51-6493-4077-ab89-fcac63f14d4a.png)

**Comment by steveoh** [comment]
Interesting. So a parcel isn't only a piece of land, it could be a floor or room in a building?

**Comment by rkelson** [comment]
Not exactly. Think of it for tax purposes; that geometry represents 3 unique parcel IDs, 3 address, and 3 tax payers
![image](https://user-images.githubusercontent.com/6200941/206540237-d0b27d18-e7c6-44e3-b791-b207d89a90e1.png)

**Comment by steveoh** [comment]
Yeah that's a much better way to think about it. Taxes.

**Comment by rkelson** [comment]
yeah parcels are more than just a boundary. For me they are a boundary with a unique ID that can be linked to all sorts of attributes across multiple offices and organizations.
